dsp-role/core/src/test/scala/dsp/role/sharedtestdata/RoleTestData.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2021 - 2022 Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.role.domain
+
+import dsp.valueobjects.Id
+import dsp.valueobjects.Permission
+import dsp.valueobjects.Role._
+
+/**
+ * Contains shared role test data.
+ */
+object RoleTestData {
+  val id1          = Id.RoleId.make()
+  val name1        = LangString.make("Name", "en")
+  val description1 = LangString.make("Description", "en")
+  val users1       = List(RoleUser(Id.UserId.make().fold(e => throw e.head, v => v)))
```

Comment:
As it is a list of users, I would add at least 2 users to it.

Reply:
IMO in this basic setup most important is to have non-empty list and if it is one, two or more users, it doesn't matter. Especially that the `Role` entity is not finalized, so the user property isn't clear how it's going to be like.